### ISSUE TITLE
refactor: remove Box3 from LoadSpec

### ISSUE THREAD

**Issue by toloudis** [allen-cell-animated/volume-viewer#254]
## Use Case

convert LoadSpec's one Box3 field and get rid of workers/utils.ts entirely

## Solution

convert the Box3 to simple array types and then the worker doesn't have to re-create classes when moving data across threads.

**Comment by ShrimpCryptid** [comment]
Waiting on a major version feature branch

**Comment by ShrimpCryptid** [comment]
Dan: Is this still in review? Would anything break if merged?
Cameron: Yes, would require major bump + bump in vole-app. Other tickets in the pipeline that would also be breaking changes (Replace old API with new event model, etc.)
Dan: Can we make a new ticket in vole-app to capture all the things that need to be updated at once? (V5 ticket)
